{{cookiecutter.project_slug}}/src/config/crowdboticsTheme.js

```diff
@@ -1,6 +1,3 @@
-import { dark, light } from "@eva-design/eva";
-
 export const crowdboticsTheme = {
-  ...light
   //todo add custom colors here
 };
```

Comment:
Is this file used anymore? Should it be removed entirely?

Reply:
it will be used in future when we defined CB theme. For now, we can remove it entirely